lib/articles_csv.rb

```diff
@@ -23,7 +23,7 @@ def header
   def data
     @object.each do |o|
       yield [
-        '',
+        o.availability ? I18n.t('simple_form.yes') : I18n.t('simple_form.no'),
```

Comment:
It would be good if the import would recognize this (I vaguely recall `x` or empty) - did you check it corresponds?

Reply:
> I vaguely recall `x` or empty

Not quite, `x` removes/deletes the article altogether.
I opened #949 to propose the addition of another key to import articles as `not available`. We should then use the same key in the exported CSV.

Reply:
There's anothe approach at the article import:
> When there is an "x" in the first column, the article is outlisted and will be removed.

So one can use the first column to mass delete articles of a supplier. The import doesn't process any available column.

Reply:
@kidhab I see you merged it now although the import behavior does not correspond (yet).
I also see an issue with the translated yes/no: This would mean the import would have to check all yes/no locales for each availability cell (or else the imported CSV would have to be in the user's locale, which could be a problem if one user exports it & another one imports it, and also for Lieferscraping).
Did you consider my proposal #949 ?

Reply:
At the moment the behaviour of the article's import and export functionality differs because they fulfill another purpose. This MR add missing feature based on the current purpose of the export functionality. And from my point of view it shouldn't take years to implement it.

Your proposal addresses a slightly different aspect of the Foodsoft. I think this is something that should be implemented. But this shouldn't block such small changes.